src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -451,10 +451,17 @@ private void visitEnumType(EnumType type) {
       emitBreak();
     }
 
-    private void visitTypeDeclaration(JSType type) {
+    private void visitTypeDeclaration(JSType type, boolean isVarArgs) {
       if (type != null) {
         emit(":");
-        visitType(type);
+        if (isVarArgs && !type.isAllType() && !type.isUnknownType()) {
```

Comment:
It just looks wacky to surround everything with extra parens. I'm hoping to emit fewer parens if possible without making our code too complex